services/py-api/src/database/repository/participants_repository.py

```diff
@@ -57,3 +58,10 @@ async def create(
         except Exception as e:
             LOG.exception("Participant insertion failed due to err {}".format(e))
             return Err(e)
+
+    async def get_number_random_participants(self):
+        return self._collection.find({"is_admin" : "false"}).count()
```

Comment:
This is now implemented in the main branch. Check 8638cbce50fa6f3458eba303a5e797c229a81b93. Pull the changes from the main branch to the branch that you are working. If you don't know how to do that just quickly google it. Resolve any conflicts if needed, you can also learn how to do that online.

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4